frontend/public/components/sidebars/resource-sidebar.tsx

```diff
@@ -29,7 +29,7 @@ const ResourceSidebarWrapper: React.FC<{
 
   return (
     <div
-      className="co-p-has-sidebar__sidebar co-p-has-sidebar__sidebar--bordered hidden-sm hidden-xs"
+      className="co-p-has-sidebar__sidebar co-p-has-sidebar__sidebar--bordered pf-v6-u-display-none pf-v6-u-display-block-on-sm"
```

Comment:
this seems working well

https://github.com/user-attachments/assets/a0afe825-d24b-454a-94e5-0b04d0a04f4d